### PR TITLE
update llama 3.3 versatile

### PR DESCRIPTION
**Pull request by sinanazeri**
## Description

adding llama 3.3 to the models. 

## Checklist

- [ x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/groq/src/hook/utils.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export const allModels = [
   { name: "Follow global model", id: "global" },
   { name: "Llama3.1 8B 132k", id: "llama-3.1-8b-instant" },
   { name: "Llama3.1 70B 132k", id: "llama-3.1-70b-versatile" },
+  { name: "Llama3.3 70B Versatile", id: "llama-3.3-70b-versatile" },
```
Comment (maximilianzuern) [inline extensions/groq/src/hook/utils.ts:7]:
```suggestion
```

File: extensions/groq/src/hook/utils.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const allModels = [
   { name: "Follow global model", id: "global" },
   { name: "Llama3.1 8B 132k", id: "llama-3.1-8b-instant" },
   { name: "Llama3.1 70B 132k", id: "llama-3.1-70b-versatile" },
```
Comment (maximilianzuern) [inline extensions/groq/src/hook/utils.ts:6]:
```suggestion
  { name: "Llama3.3 70B 132k", id: "llama-3.3-70b-versatile" },
```

File: extensions/groq/src/hook/utils.ts [hunk 3]
```diff
@@ -34,6 +35,7 @@ export function estimatePrice(prompt_token: number, output_token: number, model:
       break;
     case "llama3-70b-8192":
     case "llama-3.1-70b-versatile":
+    case "llama-3.3-70b-versatile":
```
Comment (maximilianzuern) [inline extensions/groq/src/hook/utils.ts:38]:
```suggestion
```

File: extensions/groq/src/hook/utils.ts [hunk 4]
```diff
@@ -34,6 +35,7 @@ export function estimatePrice(prompt_token: number, output_token: number, model:
       break;
     case "llama3-70b-8192":
     case "llama-3.1-70b-versatile":
```
Comment (maximilianzuern) [inline extensions/groq/src/hook/utils.ts:37]:
```suggestion
    case "llama-3.3-70b-versatile":
```

**Comment by maximilianzuern** [comment]
Since Groq has already set Llama 3.1 70B as deprecated, I prefer to just exchange the 3.1 with the 3.3 model.

**Comment by sinanazeri** [comment]
That makes sense. We can replace with "llama-3.3-70b-versatile". 
Thanks for the extension btw. I use it alot

**Review by maximilianzuern — CHANGES_REQUESTED** [review]
Please exchange the `Llama 3.1 70B` with the `Llama 3.3 70B` model in the `package.json` file as well.

**Comment by pernielsentikaer** [comment]
Hi @sinanazeri 👋 

Could you check the CI errors and also check the comments from @maximilianzuern

**Comment by orliesaurus** [comment]
> Hi @sinanazeri 👋
> 
> Could you check the CI errors and also check the comments from @maximilianzuern

Thanks for following up on this! 
P.S. love your new profile picture and new bearded look

**Comment by pernielsentikaer** [comment]
😊

**Comment by sinanazeri** [comment]
> Hi @sinanazeri 👋
> 
> Could you check the CI errors and also check the comments from @maximilianzuern

Sorry for my late reply. Sure I will fix it soon. I need to run Prettier 3.2.5

**Comment by sinanazeri** [comment]
> > Hi @sinanazeri 👋
> > Could you check the CI errors and also check the comments from @maximilianzuern
> 
> Thanks for following up on this! P.S. love your new profile picture and new bearded look

Thanks, it is heartwarming :)

**Comment by maximilianzuern** [comment]
@sinanazeri 
I already opened a PR to your `patch-llama3.3` branch  which fixes the lint issue and updates the model names 🙏 

-> https://github.com/sinanazeri/extensions/pull/1

**Review by maximilianzuern — APPROVED** [review]

**Comment by maximilianzuern** [comment]
@pernielsentikaer 
PR is ready to merge 🥳

**Review by pernielsentikaer — APPROVED** [review]
Merged 🙂

**Pull request merged**